tests/foreman/ui/test_contentview.py

```diff
@@ -2253,31 +2256,236 @@ def test_positive_clone_within_diff_env(self):
             self.assertIsNotNone(self.content_views.wait_until_element(
                 common_locators['alert.success_sub_form']))
 
-    @stubbed()
+    @run_in_one_thread
     @run_only_on('sat')
+    @skip_if_not_set('fake_manifest')
     @tier2
-    def test_positive_subscribe_system(self):
-        # Notes:
-        # this should be limited to only those content views
-        # to which you have permission, but there are/will be
-        # other tests for that.
-        # Variations:
-        # * rh content
-        # * rh custom spins
-        # * custom content
-        # * composite
-        # * CVs with puppet modules
-        """attempt to subscribe systems to content view(s)
+    def test_positive_subscribe_system_with_rh_custom_spin(self):
+        """Attempt to subscribe systems to content view(s)
```

Comment:
We need to have that string different per each to test case to represent actual actions that we perform